lib/harvesting/models/project_task_assignments.rb

```diff
@@ -0,0 +1,40 @@
+module Harvesting
+  module Models
+    class ProjectTaskAssignments < Base
+      include Harvesting::Enumerable
+      extend Forwardable
+
+      attributed :id,
```

Comment:
I don't think you these attributes here... they are relevant for the harvest records themselves, not for these collection classes. Here, you want things like `total_pages`, `previous_page`, etc.